azure-pipelines/templates/stages/build.yml

```
@@ -79,4 +79,23 @@ stages:
       displayName: Pagkage executables
 
     - publish: $(Build.ArtifactStagingDirectory)/package-$(target)
-      artifact: package-$(target)
\ No newline at end of file
+      artifact: package-$(target)
+
+    - job: PublishScripts
+      dependsOn: Build
+      condition: succeeded()
+      steps:
+        - checkout: self
+        - script: |
+            echo Copying installation scripts to artifacts directory...
+            cp ./scripts/install.cmd $(Build.ArtifactStagingDirectory)/scripts/
+            cp ./scripts/install.sh $(Build.ArtifactStagingDirectory)/scripts/
+            echo Copying uninstallation scripts to artifacts directory...
+            cp ./scripts/uninstall.cmd $(Build.ArtifactStagingDirectory)/scripts/
+            cp ./scripts/uninstall.sh $(Build.ArtifactStagingDirectory)/scripts/
+            echo Copying test scripts to artifacts directory...
+            cp ./scripts/test.cmd $(Build.ArtifactStagingDirectory)/scripts/
+            cp ./scripts/test.sh $(Build.ArtifactStagingDirectory)/scripts/
+          displayName: 'Copy scripts to artifact staging'
+        - publish: $(Build.ArtifactStagingDirectory)/scripts
+          artifact: scripts
```

The indentation of the whole jobs step is wrong 😢 very classic yaml pipeline stuff 😂 it really is a kind of a horrible format

```suggestion
  - job: PublishScripts
    dependsOn: Build
    condition: succeeded()
    steps:
      - checkout: self
      - script: |
          echo Copying installation scripts to artifacts directory...
          cp ./scripts/install.cmd $(Build.ArtifactStagingDirectory)/scripts/
          cp ./scripts/install.sh $(Build.ArtifactStagingDirectory)/scripts/
          echo Copying uninstallation scripts to artifacts directory...
          cp ./scripts/uninstall.cmd $(Build.ArtifactStagingDirectory)/scripts/
          cp ./scripts/uninstall.sh $(Build.ArtifactStagingDirectory)/scripts/
          echo Copying test scripts to artifacts directory...
          cp ./scripts/test.cmd $(Build.ArtifactStagingDirectory)/scripts/
          cp ./scripts/test.sh $(Build.ArtifactStagingDirectory)/scripts/
        displayName: 'Copy scripts to artifact staging'
      - publish: $(Build.ArtifactStagingDirectory)/scripts
        artifact: scripts
```